robocop/checkers/documentation.py

```diff
@@ -126,12 +126,19 @@ def visit_File(self, node):  # noqa
                 self.report("missing-doc-suite", node=node, lineno=1, col=1)
         super().visit_File(node)
 
-    def check_if_docs_are_present(self, node, msg):
+    def check_if_docs_are_present(self, node, msg, extend_disablers):
         for statement in node.body:
             if isinstance(statement, Documentation):
                 break
         else:
+            ext_disablers = (node.lineno, node.end_lineno) if extend_disablers else None
             if hasattr(node, "name"):
-                self.report(msg, name=node.name, node=node, end_col=node.col_offset + len(node.name) + 1)
+                self.report(
+                    msg,
+                    name=node.name,
+                    node=node,
+                    end_col=node.col_offset + len(node.name) + 1,
+                    ext_disablers=ext_disablers,
```

Comment:
`ext_disablers` name can be confusing... `ext` can be understood as `extended`, `external` or `extensions`. Better to give a full name here, I think.